con-master/src/main/java/ba/codecentric/master/controller/RoomController.java

```diff
@@ -22,5 +28,17 @@ public RoomController(RoomService roomService) {
     @PostMapping(value = "/rooms")
     public void save(@Valid @RequestBody Room room) {
         roomService.saveRoom(room);
+        LOG.info("Save Room with number: " + room.getNumber());
+    }
+
+    @GetMapping(value = "/rooms")
```

Comment:
I see that each method contains mapping value "/rooms". Maybe you should annotate controller with `@RequestMapping("/rooms")` to declutter that.